Shared/BetterColorPicker.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using BepInEx;
+using BepInEx.Configuration;
+using ChaCustom;
+using HarmonyLib;
+using Illusion.Component.UI.ColorPicker;
+using IllusionUtility.GetUtility;
+using KKAPI.Utilities;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+using PluginInfo = BetterColorPicker.Info;
+
+namespace BetterColorPicker
+{
+    [BepInDependency(KKAPI.KoikatuAPI.GUID)]
+    [BepInPlugin(PluginInfo.GUID, "Better Color Picker", PluginInfo.Version)]
+    [BepInProcess(Constants.MainGameProcessName)]
+    public class BetterColorPicker : BaseUnityPlugin
+    {
+        //public const string GUID = "marco.better_color_picker";
+        //public const string Version = "2.0.2";
```

Comment:
These are required per convention, it's to allow other plugins that reference this one to add BepInDependencyAttribute that automatically tracks the correct version number and GUID. It's always on the plugin object because there can be multiple of them in an assembly (so it's not going to be easily found if it's in a different class).

Reply:
These are commented out because I move them to Info.  Do you want them here or been in info is ok.